std/complex.d

```diff
@@ -292,14 +292,14 @@ struct Complex(T)  if (isFloatingPoint!T)
             // r = lhs
             // theta = 0
             ab = lhs ^^ this.re;
-            ar = log(lhs) * this.im;
+            ar = std.math.log(lhs) * this.im;
         }
         else
         {
             // r = -lhs
             // theta = PI
-            ab = (-lhs) ^^ this.re * exp(-PI * this.im);
-            ar = PI * this.re + log(-lhs) * this.im;
+            ab = (-lhs) ^^ this.re * std.math.exp(-std.math.PI * this.im);
+            ar = std.math.PI * this.re + std.math.log(-lhs) * this.im;
```

Comment:
This was the trickiest file. Because `std.complex` has some of the exact same symbols as `std.math`

I asked myself the question, "what are the chances that std.complex will gain a definition for PI, log, etc.?", so I just went with the static import. There are other places I have no choice (example, sqrt).